rpc_state_reader/tests/sir_tests.rs

```diff
@@ -612,3 +611,47 @@ fn starknet_in_rust_test_case_tx_skip_nonce_check(hash: &str, block_number: u64,
         }
     }
 }
+
```

Comment:
This test is great! But we need to remove the corelib files and the scarb related files.